src/ClientModel.cpp

```diff
@@ -99,7 +99,7 @@ void ClientModel::init(QString _dbpath)
 	if (m_dbpath.isEmpty())
 		m_client.reset(new MixClient(QStandardPaths::writableLocation(QStandardPaths::TempLocation).toStdString()));
 	else
-		m_client.reset(new MixClient(m_dbpath.toStdString()));
+		m_client.reset(new MixClient(QStandardPaths::writableLocation(QStandardPaths::TempLocation).toStdString() + "/" + m_dbpath.toStdString()));
```

Comment:
Probably no need to add m_dbpath here, better remove _dbpath altogether.
Also, tmp dir should be cleaned up in destructor